### PR TITLE
Parsoid: Do not query for html in case rerender is disabled

### PR DESCRIPTION
**Pull request by Pchelolo**
Finally found the reason of EMERGENCY 'unhandled error' log messages. When the the serenader is disabled, we were still making a `get` request for html content, but didn't return that promise. But the request was made anyway, so if it failed, we got an unhandled rejection. And the result is ignored anyway, so we should check if the serenader is disabled prior to making the request for html.

cc @wikimedia/services

### CODE REVIEW

File: sys/parsoid.js [hunk 1]
```diff
@@ -440,25 +441,25 @@ PSP.getFormat = function(format, restbase, req) {
         }
     }
 
+    if (!self._okayToRerender(req)) {
```
Comment (gwicke) [inline sys/parsoid.js]: This drops the no-cache check, rendering those pages completely unaccessible. Could you reinstate that check?

Comment (Pchelolo) [inline sys/parsoid.js]: @gwicke See line 402

Comment (gwicke) [inline sys/parsoid.js]: I see, missed that. I still think it would be more obvious to avoid splitting the no-cache logic, but it'll work as-is.

Comment (Pchelolo) [inline sys/parsoid.js]: If we don't split the `no-cache` logic, we'll need to duplicate the 
```
var contentReq = restbase.get({
          uri: self.getBucketURI(rp, format, rp.tid)});
});
```
code. And this check is a temporary solution, so I think its better to have all it's logic self-contained.

**Comment by gwicke** [comment]
@Pchelolo: Okay, merging.

**Pull request merged**